pyproject.toml

```diff
@@ -7,7 +7,7 @@ name = "mcp-claude-code"
 version = "0.1.27"
 description = "MCP implementation of Claude Code capabilities"
 readme = "README.md"
-requires-python = ">=3.13"
+requires-python = ">=3.12"
```

Comment:
I couldn't find any reason for 3.13 to be necessary, 3.12 worked perfectly fine.